CHANGES.md

```diff
@@ -1,6 +1,18 @@
+## Version 1.0.0
+
+> 2017 Dec 25
+
+### documentations
```

Comment:
In prior releases this section is titled **Documentation Improvements/Additions:**

Reply:
fixed.